### PR TITLE
Update llama2-model.libsonnet with training using SPMD

### PR DESCRIPTION
**Pull request by ManfeiBai**
# Description

add training llama2 with SPMD on v4-8

# Tests

**Instruction and/or command lines to reproduce your tests:** 
`./scripts/run-oneshot.sh -t pt-nightly-llama2-t-train-spmd-func-v4-8-1vm`
and tested log: http://shortn/_Jc96NxA1bK

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: tests/pytorch/nightly/llama2-model.libsonnet [hunk 1]
```diff
@@ -114,12 +114,6 @@ local utils = import 'templates/utils.libsonnet';
         # install tokenizer model
         wget https://storage.googleapis.com/tpu-pytorch/lsiyuan-experiment/llama/spiece.model
 
-        # git clone and build llama
-        git clone --branch llama2-google-next-inference https://github.com/pytorch-tpu/llama.git
```
Comment (miladm) [inline tests/pytorch/nightly/llama2-model.libsonnet:118]: I understand we don't need this cloned for training. Where do you clone llama2 for inference?

**Review by yeounoh — APPROVED** [review]
Lgtm

File: tests/pytorch/nightly/llama2-model.libsonnet (hunk 1 above)
Comment (ManfeiBai) [reply tests/pytorch/nightly/llama2-model.libsonnet:118]: we clone llama2 for inference in pjrt setup: https://github.com/ManfeiBai/ml-testing-accelerators/blob/96a8320b941d0bf9023657bd2e18ea8fedd03247/tests/pytorch/nightly/llama2-model.libsonnet#L69

**Review by miladm — APPROVED** [review]
seems the configs run locally - approving to unblock your online experimentations.

**Comment by ManfeiBai** [comment]
> seems the configs run locally - approving to unblock your online experimentations.

thanks, local tested log is http://shortn/_Jc96NxA1bK

**Pull request merged**